### PR TITLE
feature/status-error

### PR DESCRIPTION
**Pull request by jenbutongit**
**changes made**
- unknown ref check 

actually should be a `bug/` lol nvm

### CODE REVIEW

File: server/plugins/applicationStatus.js [hunk 1]
```diff
@@ -79,7 +80,11 @@ const applicationStatus = {
               await Promise.all(outputPromises)
             }
             await cacheService.clearState(request)
-            return h.view('confirmation', { reference: newReference || reference, paySkipped: userCouldntPay })
+            if (reference !== 'UNKNOWN' || newReference !== 'UNKNOWN') {
```
Comment (superafroman) [inline server/plugins/applicationStatus.js]: Should be `&&`?

Comment (jenbutongit) [reply server/plugins/applicationStatus.js]: neither(?) i guess we already deal with reference at the start of the handler

Comment (jenbutongit) [reply server/plugins/applicationStatus.js]: although would be `||` if we didn't return in the if(reference) statement. ref from cache might be unknown but newRef might not be. (context passed to template would still be wrong though)

Comment (superafroman) [reply server/plugins/applicationStatus.js]: problem is the next line - if reference is unknown and neewReference isn't, the next line will use 'UNKNOWN'

Comment (jenbutongit) [reply server/plugins/applicationStatus.js]: (outdated) next commit doesn't use reference at all now. Just does a check on newReference. Reference case would've been dealt with. commit -> 53cc636

**Pull request merged**